src/NuGet.Core/NuGet.Build.Tasks.Pack/Strings.resx

```diff
@@ -125,6 +125,10 @@
     <value>The assets file produced by restore does not exist. Try restoring the project again. The expected location of the assets file is {0}.</value>
     <comment>{0} is the path to the assets file.</comment>
   </data>
+  <data name="DefaultSymbolsPackageFormatChanging" xml:space="preserve">
+    <value>The default symbols package format will change to snupkg in an upcoming version. Use SymbolPackageFormat to keep using symbols.nupkg.</value>
```

Comment:
Should we suggest "upgrading" to the snupkg format? Maybe that should be on the docs for this warning.

Reply:
Consider using the message: 'The default symbols package format will change to snupkg in an upcoming version.;